README.md

```diff
@@ -1,13 +1,19 @@
-# wordcel
+# Wordcel
 
-Core Protocol
+Core Protocol, comprising of on-chain programs, IDL and tests.
+## Account model 
+ ![worcel accounts model](https://github.com/harsh4786/wordcel/blob/master/wordcel_accounts.png)
+  
 
-
-## Users can
+## The program is modular and can be used to create generic content not limited to written content.
 
```

Comment:
Can you talk about random_hash in this and why it is required?

Reply:
sure, just give me a moment

Reply:
done!

Reply:
lmk if this works